astro/src/content/docs/apis/identity-providers/linkedin.mdx

```diff
@@ -32,16 +32,26 @@ export const idp_linking_strategy = 'LinkByEmail';
 This API has been available since 1.23.0
 </Aside>
 
-The LinkedIn identity provider type will use OAuth 2.0 to authenticate a user with LinkedIn. It will also provide a `Login with LinkedIn` button on FusionAuth's login page that will direct a user to the LinkedIn login page. Additionally, after successful user authentication, this identity provider will call LinkedIn's `/v2/me` and `/v2/emailAddress` APIs to load additional details about the user and store them in FusionAuth.
+The LinkedIn identity provider type will use OAuth 2.0 to authenticate a user with LinkedIn. It will also provide a `Login with LinkedIn` button on FusionAuth's login page that will direct a user to the LinkedIn login page. Additionally, after successful user authentication, this identity provider will either call LinkedIn's `/v2/userinfo` API or their `/v2/me` and `/v2/emailAddress` APIs to load additional details about the user and store them in FusionAuth.
 
-The email address returned by the LinkedIn `/v2/emailAddress` API will be used to create or look up the existing user. Additional claims returned by LinkedIn can be used to reconcile the User to FusionAuth by using a LinkedIn Reconcile lambda. Unless you assign a reconcile lambda to this provider, only the `email` address will be used from the available claims returned by LinkedIn.
+The email address returned by the LinkedIn API will be used to create or look up the existing user. Additional claims returned by LinkedIn can be used to reconcile the User to FusionAuth by using a LinkedIn Reconcile lambda. Unless you assign a reconcile lambda to this provider, only the `email` address will be used from the available claims returned by LinkedIn.
 
 <TokenStorageNote idp_long_lived_token_type="access_token"
                   idp_display_name={idp_display_name}
                   idp_linking_strategy={idp_linking_strategy}
                   token_name="access_token"
                   return_text="login endpoint" />
 
+<Aside type="caution">
+  Depending on when you created your application in LinkedIn you may need to use a different set of scopes. LinkedIn had an older "compliance" program for signing in with LinkedIn that used their [Profile API](https://learn.microsoft.com/en-us/linkedin/shared/integrations/people/profile-api). The newer program is "Sign In with LinkedIn using OpenID Connect" and uses their [UserInfo API](https://learn.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/sign-in-with-linkedin-v2#api-request-to-retreive-member-details) to query for user details.
```

Comment:
Double quotes get lost here, at least in the bright Aside. Maybe backtick "compliance" and "Sign In with LinkedIn" instead of double quoting.